plugins/cli/src/parser.rs

```diff
@@ -79,7 +79,11 @@ impl Matches {
 ///     Ok(())
 ///   });
 /// ```
-pub fn get_matches(cli: &Config, package_info: &PackageInfo) -> crate::Result<Matches> {
+pub fn get_matches(
+    cli: &Config,
+    package_info: &PackageInfo,
+    args: Option<Vec<String>>,
+) -> crate::Result<Matches> {
```

Comment:
@Legend-Master Curious and slightly bored here, hope you don't mind me asking. You called out the changes to `Cli.matches` as a breaking change, how is this also not considered a breaking change? Both `Cli.matches` and `parser::get_matches` are public, and neither are referenced anywhere in the usage guide.

Reply:
`parser` is a private module, so `parser::get_matches` is not actually public, yeah I know it's kinda confusing 😂